python/src/opendp/extras/polars/__init__.py

```diff
@@ -1065,7 +1070,10 @@ def release(self) -> OnceFrame:
             raise ImportError(ERR_MSG)
 
         def summarize(self, alpha: float | None = None):
-            """Summarize the statistics released by this query."""
+            """Summarize the statistics released by this query.
+            
+            :param alpha:
```

Comment:
```suggestion
            :param alpha: optional. A value in [0, 1] denoting the statistical significance.
```

Reply:
Since `1 - alpha` comes up frequently, I'll add that here as well.